src/constructs/cloudwatch/__snapshots__/ec2-alarms.test.ts.snap

```diff
@@ -12,6 +12,20 @@ Object {
       },
     },
   },
+  "Outputs": Object {
+    "ApplicationLoadBalancerTestingApplicationLoadBalancerTestingdnsName79AE3FFF": Object {
+      "Description": "DNS entry for ApplicationLoadBalancerTesting",
+      "Export": Object {
```

Comment:
I'm not sure that we need to use [exports](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-stack-exports.html) to support either of the scenarios you've mentioned. Do you think this will be helpful for other use-cases?

Reply:
This was mainly because the logicalId is auto-generated, which made testing difficult.

I think we can safely override the logicalId for the output, not least because the auto-generated one is _really_ confusing now that I look at it! I've added a commit to do this.